base/server/python/pki/server/deployment/scriptlets/selinux_setup.py

```diff
@@ -53,6 +53,15 @@ def restore_context(self, mdict):
         selinux.restorecon(mdict['pki_instance_log_path'], True)
         selinux.restorecon(mdict['pki_instance_configuration_path'], True)
 
+    # Helper function to check if a given `context_value` exists in the given
+    # set of `records`. This method can process both port contexts and file contexts
+    def is_exist(self, records, context_value):
```

Comment:
I'd suggest renaming this method to `exists()` or `context_exists()`, or something like that.